### ISSUE TITLE
Production builds fail with latest dart-sass

### ISSUE THREAD

**Issue by deanmarcussen** [imodeljs/imodeljs#2249]
**Describe the bug**

This line
https://github.com/imodeljs/imodeljs/blob/d0c88bd98674525ff1fe6533c463bccedcedc8e7/ui/framework/src/ui-framework/content/DefaultViewOverlay.scss#L15

produces this error on our latest prod build. Probably the version of sass is `1.41.0` (always hard to tell  on devops)
```
SassError: "+" and "-" must be surrounded by whitespace in calculations.
   ╷
15 │   width: calc(100% -80px);
   │                    ^
```

It does compile with dart-sass `1.39.0` on my dev machine, so have fixed the prod build by forcing the older version

**Comment by calebmshafer** [comment]
@deanmarcussen thanks for reporting this. This #2239 should fix the issue.  We'll be backporting it back to 2.19.x and releasing a patch for it later today.

**Comment by aruniverse** [comment]
Should be fixed in 2.19.12, which is now available.
